src/rules/missing-playwright-await.ts

```diff
@@ -57,6 +57,76 @@ const playwrightTestMatchers = [
   'toBeInViewport',
 ]
 
+const pageMethods = new Set([
```

Comment:
Please double check these 🙏 I tried to alphabetically sort them and make sure I included all methods that return `Promise` and not any that don't, but I could have made a mistake here.